lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/metrics/DurationTiming.java

```diff
@@ -34,7 +34,7 @@
     @JsonCreator
     public DurationTiming(Duration duration)
     {
-        this.duration = requireNonNull(duration, "duration is null");
+        this.duration = requireNonNull(duration, "duration is null").convertToMostSuccinctTimeUnit();
```

Comment:
This conversion is “lossy”, so that’s probably not what we want. If it’s for the purpose of displaying a more human friendly version, it should be done at rendering time.

Reply:
Good point. I did not know that. 

FWIW I see there are many places where we convert early still. E.g. https://github.com/trinodb/trino/blob/43704ae4ca1de324074fc8cc5334f3cb13c0e025/core/trino-main/src/main/java/io/trino/operator/DriverContext.java#L413-L415

Reply:
@martint we use `convertToMostSuccinctTimeUnit` when creating and merging `io.trino.operator.OperatorStats` (and stats above it). It serves 2 purposes:
* display stats nicely in `EXPLAIN ANALYZE`
* serialize stats nicely (and in same way as other timing stats) in query JSON.

Reply:
I understand, but serialization format is actually not a goal (the query json is not meant to be human readable). Correcting this is something we have been thinking about for some time. We already did so for DataSize. cc @pettyjamesm

Reply:
chagned